_data/release_notes_versions.json

```diff
@@ -1456,5 +1456,23 @@
       ],
       "api-docs":"ballerina-api-docs-1.2.53.zip",
       "release-notes":"ballerina-release-notes-1.2.53.md"
-   }
+   },
+   {
+        "version":"1.2.54",
+        "release-date":"2024-07-26",
+        "windows-installer":"ballerina-windows-installer-x64-1.2.54.msi",
+        "windows-installer-size":"272K",
+        "linux-installer":"ballerina-linux-installer-x64-1.2.54.deb",
+        "linux-installer-size":"152mb",
+        "macos-installer":"ballerina-macos-installer-x64-1.2.54.pkg",
+        "macos-installer-size":"172mb",
+        "rpm-installer":"ballerina-linux-installer-x64-1.2.54.rpm",
+        "rpm-installer-size":"175mb",
+        "other-artefacts":[  
+            "ballerina-1.2.54.zip",
+            "ballerina-1.2.54.vsix"
```

Comment:
We can remove the vsix artifact as we are not using it anymore.

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/9178/commits/c212d2eeaa1fc1be14ecbec925f939d52ff34ef9